src/components/ReviewCard.jsx

```diff
@@ -1,8 +1,19 @@
-const ReviewCard = () => {
+const ReviewCard = ({ review }) => {
   return (
-    <div>
-      <h3>Review card title</h3>
-      <p>more info about the review</p>
+    <div className="ReviewCard">
+      <div className="imgContainer">
+        <img
+          className="reviewImg"
+          src={review.review_img_url}
+          alt={review.title}
+        />
+      </div>
+
+      <h3 className="reviewTitle">
+        {review.title.length > 50
```

Comment:
This is neat... you could achieve a similar thing with CSS, look at text-overflow